server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/template_configs/approval_widget.js.msx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2016 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+define([
+    'mithril', 'string-plus', 'jquery', 'helpers/form_helper', 'helpers/pipeline_configs/tooltips',
+    'helpers/mithril_component_mixins', 'models/shared/users', 'models/shared/roles', 'jquery.textcomplete'
+], function (m, s, $, f, tt, ComponentMixins, Users, Roles) {
+
+  var autoComplete = function (values, model, attrName) {
```

Comment:
autoComplete method is already implemented as a part of ```views/pipeline_configs/stages_config_widget``` if possible can you please reuse the same method ?